### PR TITLE
Add support for commonjs

### PR DESCRIPTION
**Pull request by Pipo93**
This addresses issue https://github.com/adrienjoly/npm-pdfreader/issues/133

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +14,8 @@
     "test:update": "ava --update-snapshots",
     "test": "ava",
     "lint": "eslint .",
-    "semantic-release": "semantic-release"
+    "semantic-release": "semantic-release --dry-run",
```
Comment (adrienjoly) [inline package.json:17]: Can we revert that change please?

```suggestion
    "semantic-release": "semantic-release",
```

**Review by adrienjoly — APPROVED** [review]
Overall, LGTM.

Can you just apply (or challenge) the suggested change before we merge, please?

Thank you for your contribution!

File: package.json (hunk 1 above)
Comment (Pipo93) [reply package.json]: My mistake, fixed it :)

**Review by adrienjoly — APPROVED** [review]
LGTM. merging

**Pull request merged**